### DISCUSSION TITLE
Select the folders properly

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#22] [Ideas]
It would be really nice, even to select the folders properly and not using only the Mobius-Folder

_Originally posted by @rokondo in https://github.com/MobiusSync/MobiusSync/issues/9#issuecomment-761806157_

**Comment by SimonPickup** [comment]
> It would be really nice, even to select the folders properly and not using only the Mobius-Folder
> 
> _Originally posted by @rokondo in [#9 (comment)](https://github.com/MobiusSync/MobiusSync/issues/9#issuecomment-761806157)_

@rokondo, I'm not exactly what you mean by this. If you mean you'd like to be able to select for sync any of the folders visible under _Files_ app, this is not possible under iOS due to privacy constraints.

(Moved into Discussions as this was presumably not related to Issue #9)

**Reply by rokdd** [reply]
Theoretically on my Android phone I share the folder with gpx log files and the photo folder with my devices. On my iphone I would only need this for the camera folder.

**Reply by SimonPickup** [reply]
OK, as you say the camera folder will be handled specifically via the API for accessing photos/videos.
The use case for GPX log files won't work on iOS as another app cannot save into Möbius Sync's folders.

**Reply by SimonPickup** [reply]
Actually other apps could implement this using the same was as a-Shell, i.e. to save/access their files within Möbius Sync's sandbox.

**Comment by Barobi** [comment]
> It would be really nice, even to select the folders properly and not using only the Mobius-Folder

They did it here: https://github.com/holzschu/a-shell#sandbox-and-bookmarks
Possible since iOS 13.

**Comment by SimonPickup** [comment]
Created #59.
